api/server/models/user.js

```diff
@@ -280,4 +281,49 @@ module.exports = function (User) {
 //********************************* END RESET PASSWORD **********************************
 
 
+//********************************* START FORGET PASSWORD **********************************
+    User.forget_password = function (email_id, callback) {
+        email_id = email_id.toLowerCase();
+        where = {
+            email_id: email_id
+        };
+        User.find({where: where}, function (err, result) {
+            if (err) {
+                callback(null, 0, err, {});
+            } else {
+                if (result.length == 0) {
+                    callback(null, 'Email Id not exists');
```

Comment:
even if email doesn't exist put this message 
You should get a new password on your email address, if you have an account with us